create/create_test.go

```diff
@@ -11,7 +11,18 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+type dummyRunner struct{}
+
+func (r dummyRunner) Run() error { return nil }
```

Comment:
Can we add a quick test to assert that the runner runs when calling create?

Reply:
Done!